src/routes/RequestsRoute.js

```diff
@@ -115,9 +115,9 @@ class RequestsRoute extends React.Component {
         params: {
           query: makeQueryFunction(
             'cql.allRecords=1',
-            '(requesterId=="%{query.query}" or requester.barcode="%{query.query}*" or item.title="%{query.query}*" or item.barcode=="%{query.query}*" or itemId=="%{query.query}" or itemIsbn="%{query.query}")',
+            '(requesterId=="%{query.query}" or requester.barcode="%{query.query}*" or instance.title="%{query.query}*" or item.barcode=="%{query.query}*" or itemId=="%{query.query}" or itemIsbn="%{query.query}")',
```

Comment:
Should `itemId=="%{query.query}"` remain here, or should it be replaced by/supplemented by `instanceId=="%{query.query}"`

Reply:
`itemId` will be at the same place after migration. probably we need to add instanceId.

Reply:
Done. Added `instanceId` into query string.